include/re_tls.h

```diff
@@ -63,6 +63,7 @@ int tls_set_verify_server(struct tls_conn *tc, const char *host);
 
 int tls_get_issuer(struct tls *tls, struct mbuf *mb);
 int tls_get_subject(struct tls *tls, struct mbuf *mb);
+void tls_enable_sverify(struct tls *tls, bool enable);
```

Comment:
I think `tls_disable_server_verify(struct tls *tls)` and if needed `tls_enable_server_verify(struct tls *tls)` would be more clear.

Reply:
maybe more consistent would be `tls_disable_verify_server`